src/sentry/api/serializers/base.py

```diff
@@ -12,7 +12,9 @@ def serialize(objects, user=None, serializer=None):
 
     if not objects:
         return objects
-    elif not isinstance(objects, (list, tuple)):
+    # sets aren't predictable, so generally you should use a list, but it's
+    # supported out of convenience
+    elif not isinstance(objects, (list, tuple, set, frozenset)):
```

Comment:
only added this so I could pass them in the areas where we don't care about order here -- didn't feel worth it to coerce them all to lists just to pass them into this func